src/utils/validation/index.ts

```diff
@@ -17,6 +17,15 @@ export function isValidAddress(address: string) {
   return false;
 }
 
+// basic format checking for a secret phrase
+// we want to return true if there are 12 words and return false if there are not, so we can warn the user their secret may be malformed
+// note: any secret length is acceptable by JUP standards, but if they've been generated through normal channels they will be 12 words
+export function isValidSecret(secret: string) {
+  const acceptableWordCount = 12;
+  const secretWordCount = secret.split(" ").length;
```

Comment:
are they always going to put the seed phrase in with spaces? what if they accidentally double space? or use tabs? are these edge cases you want to care about? or would it be simple enough to ensure there is text letting them know what format is expected (a little question mark button or something for help) or even just text on the seed phrase entry page.

to be clear, your logic for splitting and checking the length is perfectly adequate, unless they double space or have multiple spaces (like someone's editor adding spaces when they push tab). i can see that being the main edge case to eliminate from the input, before splitting.

Reply:
I think a tooltip is the best route, explaining the most common seed phrase format. We have to let them login no matter what they type, because they may have screwed up their seed phrase at some point and funded the screwed up account and if we prevent them from logging in we're preventing them from seeing their funds which is bad. 

Since we can't reverse engineer their mistake we can't really add much more value than saying "your seed doesn't look quite right", and telling them what a correct format is. I'll make an issue on this.